templates/microstack/index.html

```diff
@@ -0,0 +1,358 @@
+{% extends 'base_index.html' %}
+
+{% block title %}OpenStack on Kubernetes | Ubuntu{% endblock %}
+
+{% block description %}
+  MicroStack delivers distilled OpenStack excellence with native K8s experience. OCI images, Kubernetes operators, lucid interface - all you need to run fully functional OpenStack on Kubernetes with minimal effort.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <script defer src="{{ versioned_static('js/osselector.js') }}"></script>
```

Comment:
Good catch! It didn't. Added and pushed.

Reply:
just adding the file is not enough, it needs to be added to the build in order to be available for versioned_static function

Reply:
The file was being referenced directly, hence was not added to the build in the first place. I have now added the file to build, and referenced the build/minimized version of the file rather than the original.